### PR TITLE
fix: Add extra yaml files to main manifest.

### PR DESCRIPTION
**Pull request by gary-van-woerkens**

### CODE REVIEW

File: .k8s/__tests__/yaml/kosko generate --env prod.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    readdirSync: jest
+      .fn()
+      .mockReturnValue(["redirect.yaml"]),
+  }));
+  jest.doMock("@kosko/yaml", () => ({
+    loadFile: (
+      file,
+      { transform }
+    ) =>
+      jest.fn().mockResolvedValueOnce(transform(loadString(yaml)[0])),
```
Comment (douglasduteil) [inline .k8s/__tests__/yaml/kosko generate --env prod.ts:32]: Why are we mocking fs ?

File: .k8s/__tests__/yaml/kosko generate --env prod.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+//     ) =>
+//       jest.fn().mockResolvedValueOnce(transform(loadString(yaml)[0])),
+//   }));
+//   expect(
+//     await getEnvManifests("prod", "'!(_*)'", {
+//       ...project(name).prod,
+//       SOCIALGOUV_CONFIG_PATH: __dirname + "/config.json",
+//     })
+//   ).toMatchSnapshot()
+// })
```
Comment (douglasduteil) [inline .k8s/__tests__/yaml/kosko generate --env prod.ts:80]: can be removed ?

File: .k8s/babel.config.js [hunk 3]
```diff
@@ -1,3 +1,6 @@
 module.exports = {
   plugins: [["@babel/plugin-transform-modules-commonjs"]],
+  // moduleNameMapper: {
+  //   "config/stuff.yaml": "<rootDir>/config/__mocks__/stuff.yaml"
+  // }
```
Comment (douglasduteil) [inline .k8s/babel.config.js:5]: can be removed ?

File: .k8s/package.json [hunk 4]
```diff
@@ -13,6 +13,7 @@
     "@types/jest": "^26.0.20",
     "dotenv": "^8.2.0",
     "jest": "^26.6.3",
+    "jest-yaml-transform": "^0.2.0",
```
Comment (douglasduteil) [inline .k8s/package.json:16]: Where is it used ?

File: .k8s/utils/mock-directory.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  const cwd = directory();
+
+  fs.symlinkSync(path.join(__dirname, "..", "utils"), `${cwd}/utils`)
+  fsx.copy(path.join(__dirname, "..", "components"), `${cwd}/components`)
+  fs.symlinkSync(path.join(__dirname, "..", "kosko.toml"), `${cwd}/kosko.toml`)
+  fs.symlinkSync(path.join(__dirname, "..", "node_modules"), `${cwd}/node_modules`)
+  fs.symlinkSync(path.join(__dirname, "..", "tsconfig.json"), `${cwd}/tsconfig.json`)
+
+  return cwd
+}
```
Comment (douglasduteil) [inline .k8s/utils/mock-directory.ts:16]: IO action are better handled with async
You can use `fs-extra` everywhere with `await` on each fs like functions

**Review by revolunet — APPROVED** [review]

**Pull request merged**